sdk/java-sdk-spring/src/it/java/com/example/wiring/tracing/TracingIntegrationTest.java

```diff
@@ -37,20 +38,25 @@ public TracingIntegrationTest(ApplicationContext applicationContext) {
         super(applicationContext, config);
     }
 
-    @Disabled //disabled ATM for https://github.com/lightbend/kalix-jvm-sdk/pull/1810
+    @Test //disabled ATM for https://github.com/lightbend/kalix-jvm-sdk/pull/1810
```

Comment:
No longer disabled?

Reply:
That's what I was trying to make to work. Sorry, I'll delete the comment. It's confusing, ofc.

Reply:
The whole idea is to enable the test